LabZoo/Animal.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace LabZoo
+{
+    public class Animal
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello World");
+        }
+    }
+
+        public abstract class Animals
```

Comment:
Class names should nearly always be singular. I'm make a `new Animal()` not `new Animals()`.